pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angular-google-chart</artifactId>
-    <version>0.1.1-SNAPSHOT</version>
+    <version>0.1.0.1-SNAPSHOT</version>
```

Comment:
Can you make this `0.1.0-1-SNAPSHOT` since that is the convention we use elsewhere?